skiko/src/commonMain/kotlin/org/jetbrains/skia/paragraph/StrutStyle.kt

```diff
@@ -211,6 +211,7 @@ class StrutStyle internal constructor(ptr: NativePointer) : Managed(ptr, _Finali
         return this
     }
 
```

Comment:
If this PR isn't needed in Compose 1.7, let's wait merging https://github.com/JetBrains/skiko/pull/988, which we need in 1.7

Reply:
Yep, it's targeted to post-1.7 Compose

Reply:
merged, built